src/runtime/virtcontainers/pkg/oci/utils_test.go

```diff
@@ -16,8 +16,9 @@ import (
 	"strings"
 	"testing"
 
-	"github.com/cri-o/cri-o/pkg/annotations"
+	ctrAnnotations "github.com/containerd/containerd/pkg/cri/annotations"
 	crioAnnotations "github.com/cri-o/cri-o/pkg/annotations"
+	dockerAnnotations "github.com/kata-containers/kata-containers/src/runtime/virtcontainers/pkg/annotations/dockershim"
```

Comment:
I don't this file existing in out repo yet. Did you forget to include it? Also, should'nt this be an upstream change to dockershim itself? Is that something that you are working on?